packages/lit-dev-content/site/docs/v1/lit-html/getting-started.md

```diff
@@ -19,12 +19,12 @@ lit-html is distributed on npm, in the [lit-html package].
 npm install lit-html
 ```
 
-### unpkg.com
+### CDNs
 
-You can also load lit-html directly from the unpkg.com CDN:
+You can also load lit-html directly from CDNs with good module support like [esm.run](https://esm.run) or [esm.sh](https://esm.sh):
 
 ```js
-import {html, render} from 'https://unpkg.com/lit-html?module';
+import {html, render} from 'https://esm.run/lit-html';
```

Comment:
I could go either way on this. The docs are explicitly versioned so this could also be versioned. But also the newer lit-html is probably preferable, so it might be good to nudge for the newer library?

Reply:
This did specifically break in unpkg starting v3 so pinning to v1 would've prevented that. It is ultimately up to the user to choose their acceptable version pinning. It's also old doc that hopefully isn't really being visited much so it's of less significance. I lean towards pinning major though since that's default npm behavior and should de-risk future breakages.